src/views/portfolio/projects/ProjectCreateProjectModal.vue

```diff
@@ -166,30 +185,59 @@
         }).then((response) => {
           this.$emit('refreshTable');
           this.$toastr.s(this.$t('message.project_created'));
+          this.selectedParent = null;
+          this.availableParents = [{ value: null, text: ''}]
         }).catch((error) => {
           this.$toastr.w(this.$t('condition.unsuccessful_action'));
         }).finally(() => {
           this.$root.$emit('bv::hide::modal', 'projectCreateProjectModal');
         });
       },
       retrieveLicenses: function() {
-        let url = `${this.$api.BASE_URL}/${this.$api.URL_LICENSE_CONCISE}`;
-        this.axios.get(url).then((response) => {
-          for (let i = 0; i < response.data.length; i++) {
-            let license = response.data[i];
-            this.selectableLicenses.push({value: license.licenseId, text: license.name});
-            if (this.project.resolvedLicense && this.project.resolvedLicense.uuid === license.uuid ) {
-              this.selectedLicense = license.licenseId;
+        return new Promise(resolve => {
+          let url = `${this.$api.BASE_URL}/${this.$api.URL_LICENSE_CONCISE}`;
+          this.axios.get(url).then((response) => {
+            for (let i = 0; i < response.data.length; i++) {
+              let license = response.data[i];
+              this.selectableLicenses.push({value: license.licenseId, text: license.name});
+              if (this.project.resolvedLicense && this.project.resolvedLicense.uuid === license.uuid ) {
+                this.selectedLicense = license.licenseId;
+              }
             }
-          }
-        }).catch((error) => {
-          this.$toastr.w(this.$t('condition.unsuccessful_action'));
-        });
+          }).catch((error) => {
+            this.$toastr.w(this.$t('condition.unsuccessful_action'));
+          }).finally(() => {
+            resolve()
+          });
+        })
       },
       resetValues: function () {
         this.project = {};
         this.tag = "";
         this.tags = [];
+        this.selectedParent = null
+        this.availableParents = []
+      },
+      createProjectLabel: function (project) {
+        if (project.version){
+          return project.name + " : " + project.version
+        } else {
+          return project.name
+        }
+      },
+      asyncFind: function (query) {
+        if (query){
+          this.isLoading = true
+          let url = `${this.$api.BASE_URL}/${this.$api.URL_PROJECT}?searchText=${query}&excludeInactive=false`
```

Comment:
Terribly sorry, I made a mistake when I initially shared the endpoint URL with you. 😅 It should be `excludeInactive=true` in your case, so that inactive projects are not queried and returned by the API. 

But I just tested it, and other than it currently returning inactive projects, it works great! Awesome job! 🙌